themes/bootstrap3/js/common.js

```diff
@@ -63,8 +63,8 @@ var VuFind = (function VuFind() {
     }
 
     // iterate over a copy of the listeners array
-    // this prevents listeners from being skipped
-    // if the listener before it is removed during execution
+    // this prevents listeners from being skipped if
```

Comment:
This change moves the "if" from one line to another but doesn't seem to resolve the fact that this is an incomplete sentence.

Reply:
Trying to clarify what I'm hoping to express: "Iterate over a copy of the listeners array. This prevents the loop from skipping listeners if any are removed."

Reply:
Feel free to rewrite the comment if you think it makes things more clear. My point was just that if the only change is moving "if" from one line to a different line, then the change doesn't accomplish much. ;-)